usersService/src/main/java/il/ac/afeka/usersservice/boundaries/DepartmentIdBoundary.java

```diff
@@ -0,0 +1,27 @@
+package il.ac.afeka.usersservice.boundaries;
+
+public class DepartmentIdBoundary {
+
+	private String deptId;
+
+	public DepartmentIdBoundary() {}
+
+	public DepartmentIdBoundary(String deptId) {
+		this.deptId = deptId;
+	}
+
+	public String getDeptId() {
+		return deptId;
+	}
+
+	public void setDeptId(String deptId) {
+		this.deptId = deptId;
+	}
+
+	@Override
+	public String toString() {
+		return "DepartmentIdBoundary{" +
+				"depId='" + deptId + '\'' +
+				'}';
+	}
+}
```

Comment:
The fix is good, but the boundary is still lacking. 
look at the differences between what's the body structure in swagger and what Eyal have requested:
![image](https://github.com/Yuvalxa/CloudComputing/assets/90918336/41e1d201-9275-40c6-9eb8-6a9e56d95779)